Katana/Plastic/Value.swift

```diff
@@ -145,3 +187,65 @@ public struct Value: Equatable {
     return lhs.scalable == rhs.scalable && lhs.fixed == rhs.fixed
   }
 }
+
+/**
+ This allows `Value` instances to be initialized by integer and floating point literals,
+ having the scalable part equal to the numeric literal, and the fixed part equal to 0.
+ */
+extension Value: ExpressibleByIntegerLiteral, ExpressibleByFloatLiteral {
```

Comment:
`they make it clear to users of Value why using integer/float literals in operations expecting Values works.` to me, they don't make it clear. You have to discover that you have implemented that protocol somewhere to understand the code, otherwise you don't get why it works. 

If you define the proper + operator, if is oblivious the reason why it works. 

In general we decided to don't use them multiple times because they can create confusing situations..
For instance (not tested, but it should work)
```swift
init(width: Value, height: Value) -> Size

// you can do
Size(width: 5, height: 7)
```
You are basically propagating this ability to use double instead of values everywhere and to me this is not good